src/reporter.js

```diff
@@ -38,30 +35,15 @@ var resultsFolder;
 
 var uniqueFileId;
 
-function getSettings(options) {
+function getConfiguration(options) {
```

Comment:
Better keep the old `loadConfiguration` name. Notice, you are not getting (which assumes returning a value) but loading.